sources/packages/web/src/components/common/restriction/ViewRestriction.vue

```diff
@@ -63,12 +59,33 @@
                 ><title-value
                   propertyTitle="Date resolved"
                   :propertyValue="
-                    dateOnlyLongString(restrictionData.updatedAt)
+                    dateOnlyLongString(restrictionData.resolvedAt)
                   " /></v-col
               ><v-col
                 ><title-value
                   propertyTitle="Resolved by"
-                  :propertyValue="restrictionData.updatedBy" /></v-col
+                  :propertyValue="restrictionData.resolvedBy" /></v-col
+            ></v-row>
+          </content-group>
+        </template>
+        <template v-if="showDeletion">
+          <h3 class="category-header-medium mt-2">Deletion</h3>
+          <content-group v-if="!!restrictionData.deletedAt">
```

Comment:
Do we require this condition on top of condition at line 72?

Reply:
Adjusted.